src/styles/main.scss

```diff
@@ -1 +1,67 @@
-// put code here
+.body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vmin;
+  margin: 0;
+}
+
+@keyframes rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  border-radius: 50%;
```

Comment:
'border-radius: 50%;' is duplicated
please move all duplicated styles to the variables